src/app/features/public-form/public-form-routing.ts

```diff
@@ -0,0 +1,11 @@
+import { Routes } from "@angular/router";
+import { SubmissionSuccessComponent } from "./submission-success/submission-success.component";
+import { PublicFormComponent } from "./public-form.component";
+
+export const publicFormRoutes: Routes = [
+  {
+    path: "public-form/:id",
```

Comment:
```suggestion
    path: "form/:id", // added as subroute resulting in /public-form/form/:id
```

I think route changed because child-routes already have a prefix of public-form/ from the app.routing? But somehow both /public-form and public-form/public-form works ... Any idea why, @Abhinegi2 ?

Reply:
I checked and found that both /public-form and /public-form/public-form are working 🙈. 
In the PublicFormComponent, we are searching for parameters(id), and the id we get in both cases is the same (e.g., "publicformid"). 
This happens because the /public-form route is already defined in-app.routing, so the page exists regardless of the redundant prefix. That could be why both routes work.

Reply:
now works as expected, thanks!

Don't actually quite understand why not before, but whatever :sweat_smile: